frontends/infinite-corridor/src/scss/homepage.scss

```diff
@@ -28,51 +32,45 @@ $link-max-size: 120px;
   }
 
   h2 {
-    margin-top: $spacer;
-    color: theme.$color-primary;
-    text-align: center;
+    font-size: 22px;
+    max-width: 430px;
+    margin: 0;
+    font-weight: 400;
```

Comment:
There is a `theme.$font-weight-semi-bold` which might be appropriate here. It's currently 500, but I guess we removed whatever was using it, so you could change it.

_I do generally think it would better for things like fonts and colors to be controlled via `theme.scss`_